PostProcessing/Runtime/PostProcessLayer.cs

```diff
@@ -114,11 +114,11 @@ public enum Antialiasing
         [SerializeField]
         PostProcessResources m_Resources;
 
-#pragma warning disable 169
+#if UNITY_EDITOR
```

Comment:
This adds serialized fields in editor only.
As a result, when building, these fields will be serialized, but when running the build, they won't be deserialized. 
This results in very verbose errors when playing the build, about deserialized size not matching with what is expected.

Unless I'm missing something big here, this does not seem to be what you want.